src/appleseed/renderer/meta/tests/test_sss.cpp

```diff
@@ -31,6 +31,7 @@
 #include "renderer/kernel/shading/shadingpointbuilder.h"
 #include "renderer/modeling/bssrdf/bssrdf.h"
 #include "renderer/modeling/bssrdf/directionaldipolebssrdf.h"
+#include "renderer/modeling/bssrdf/normalizeddiffusionbssrdf.h"
```

Comment:
Normalized diffusion tests must be conditioned by the `APPLESEED_WITH_NORMALIZED_DIFFUSION_BSSRDF` preprocessor symbol.